### PR TITLE
Fix some flaws in the "LED roulette" section

### PR DESCRIPTION
**Pull request by SimplyDanny**

### CODE REVIEW

File: src/05-led-roulette/README.md [hunk 1]
```diff
@@ -36,8 +36,8 @@ The `no_main` attribute says that this program won't use the standard `main` int
 tailored for command line applications that receive arguments. Instead of the standard `main` we'll
 use the `entry` attribute from the [`cortex-m-rt`] crate to define a custom entry point. In this
 program we have named the entry point "main", but any other name could have been used. The entry
-point function must have signature `fn() -> !`; this type indicates that the function can't return
--- this means that the program never terminates.
+point function must have the signature `fn() -> !`; this type indicates that the function can't
+return - this means that the program never terminates.
```
Comment (adamgreig) [inline src/05-led-roulette/README.md:40]: I think the intent with `--` was to create an en-dash, so suggesting that here instead.

```suggestion
return – this means that the program never terminates.
```

**Review by adamgreig — CHANGES_REQUESTED** [review]
These are great, thanks for going through! I've just got one suggested change, the rest look good to me.

File: src/05-led-roulette/README.md (hunk 1 above)
Comment (SimplyDanny) [reply src/05-led-roulette/README.md]: I see. Changed it.

File: src/05-led-roulette/debug-it.md [hunk 2]
```diff
@@ -10,9 +10,9 @@ The starter project I've provided to you has some extra code that runs *before*
 At this time, we are not interested in that "pre-main" part so let's skip right to the beginning of
 the `main` function. We'll do that using a breakpoint. Issue `break main` at the `(gdb)` prompt:
 
-> **Note** for these gdb commands I generally won't provide a copyable code block
+> **NOTE** For these GDB commands I generally won't provide a copyable code block
```
Comment (SimplyDanny) [inline src/05-led-roulette/debug-it.md:13]: I forgot this change in my previous commit. Now it is updated accordingly. Sorry for that!

**Comment by SimplyDanny** [comment]
@adamgreig: Is there anything more I need to do to get this PR accepted?

**Review by adamgreig — APPROVED** [review]
Oh, sorry! I totally missed your subsequent commits, this all looks fine, thanks for the reminder!

bors merge

**Pull request merged**